protocol/pubsub/v2/protocol_test.go

```diff
@@ -20,6 +23,32 @@ type testPubsubClient struct {
 	conn *grpc.ClientConn
 }
 
+func (pc *testPubsubClient) NewWithAttributesInterceptor(ctx context.Context, projectID, orderingKey string) (*pubsub.Client, error) {
```

Comment:
I'm confused by the design of the PubsubClient... this PR defines `NewWithAttributesInterceptor` and below I see `NewWithOrderInterceptor`, does this really mean a user can't have Attributes and Order enabled for the same client? I must be missing something.

Reply:
That's a test file.

Reply:
doi! sorry